challenge-1/picture-of-the-day/src/components/MediaSnippet/MediaSnippet.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Image } from '../../atoms';
+
+function MediaSnippet({ url, mediaType }) {
+  if (mediaType === 'image') {
+    return (
+      <Image>
+        <img width="90%" src={url} alt="NASA APOTD" data-testid="APOTD" />
+      </Image>
```

Comment:
Even though the component works, I think there is another approach you might want to take into consideration.

<Image> could receive the props width="90%" src={url} alt="NASA APOTD" data-testid="APOTD" instead of passing the img as children. No need to change this, just have it in mind. Nice work!